nexus/reconfigurator/planning/tests/output/planner_nonprovisionable_2_2a.txt

```diff
@@ -171,11 +321,11 @@ ERRORS:
     Nexus {
         internal_address: [fd01:1122:3344:105::22]:12221,
         external_ip: OmicronZoneExternalFloatingIp {
-            id: cd63774a-2e2f-49ce-a3df-33e3b5d02650 (external_ip),
+            id: a6c185b0-b470-4631-b6c8-373349430abb (external_ip),
```

Comment:
I don't recall how this gets created. Is this changing going to be a problem for future test runs?

Reply:
I'm pretty sure it's using a "deterministic RNG", and may have shifted because I'm also using that RNG generator to make the dataset UUIDs deterministic.

This test passes without EXPECTORATE overwrite, so at it's current position, the UUID is stable.